### PR TITLE
New pages new content

### PR DESCRIPTION
**Pull request by OpenGlobe**
Breaks the rest of the NTIS pages out as separate pages for the purposes of navigation, and adds the rough draft bios and photos to the join page.  @robertsosinski @khandelwal

### CODE REVIEW

File: join.html [hunk 1]
```diff
@@ -11,16 +11,16 @@
...
   </div>
   <div class="usa-width-one-third">
     <img src="{{ site.baseurl }}/assets/site/img/headshot.jpg" />
-    <h3>Paul</h3>
-    <h4>Data Scientist</h4>
-    <p>I love working on the infrastructure and platforms to host data.</p>
+    <h3>Derssie</h3>
+    <h4>Chief Data Scientist</h4>
+    <p>Derssie Mebratu received the B.S. from McMaster university and M.Eng and Ph.D. degree in electrical engineering from Howard University. Dr. Mebratu’s research interests include Statistical signal processing, Predictive analytics, Bayesian statistical analysis, Linear dynamic model, Clustering algorithm, Queuing theory, Neural network, Semantics ontology and Text analytics. Prior to coming to NTIS, Dr. Mebratu was working as Algorithm Scientist at Orbis Technologies and as  research engineering at NASA Goddard.  Dr. Mebratu is currently working as a Chief Data Scientist at the National Technical Information Service in the Department of Commerce, to develop a cutting-edge technology in the field of Data science.</p>
```
Comment (khandelwal) [inline join.html]: I think a shorter paragraph will do much better here for Derssie. I'd drop the educational pedigree. While it's important, it's less useful in encouraging others to apply.

File: work.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: NTIS
+---
+
+<div class="usa-grid">
+  <h1>Work with us</h1>
+
+    <h2>Building a new bridge between government and private sector innovation.</h2>
```
Comment (khandelwal) [inline work.html:9]: If we're going to say this phrase - then we should explain what it means. How does it all work?

**Comment by OpenGlobe** [comment]
@emileighoutlaw - mind taking a look at Shashank's notes above?  I think these (hopefully) are easy fixes.

**Comment by OpenGlobe** [comment]
@emileighoutlaw - this still needs your magic, but can you take a quick look and merge if you think the edits (for now) address @khandelwal 's comments?  I want to flag for initial look today (we'll ask for a more formal review shortly once we get more of the revised content up).

**Comment by emileighoutlaw** [comment]
Absolutely. Taking a gander now!

**Comment by emileighoutlaw** [comment]
I think this is a good start. I _do_ want to tweak things early next week, but I feel good about merging this today

**Pull request merged**